gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -446,17 +490,22 @@ MM_VerboseHandlerOutputStandard::handleScavengeEnd(J9HookInterface** hook, uintp
 void
 MM_VerboseHandlerOutputStandard::handleConcurrentEndInternal(J9HookInterface** hook, uintptr_t eventNum, void* eventData)
 {
-	/* convert event from concurrent-end to scavenge-end */
-	MM_ScavengeEndEvent scavengeEndEvent;
-	MM_ConcurrentPhaseEndEvent *event = (MM_ConcurrentPhaseEndEvent *)eventData;
-
-	scavengeEndEvent.currentThread = event->currentThread;
-	scavengeEndEvent.timestamp = event->timestamp;
-	scavengeEndEvent.eventid = event->eventid;
-	scavengeEndEvent.subSpace = NULL; //unknown info
-	scavengeEndEvent.cycleEnd = false;
-
-	handleScavengeEndNoLock(hook, J9HOOK_MM_PRIVATE_SCAVENGE_END, &scavengeEndEvent);
+	if (J9HOOK_MM_PRIVATE_CONCURRENT_GLOBAL_PHASE_END == eventNum) {
+		handleConcurrentMarkEnd(hook, J9HOOK_MM_PRIVATE_CONCURRENT_GLOBAL_PHASE_END, eventData);
+	} else if (J9HOOK_MM_PRIVATE_CONCURRENT_LOCAL_PHASE_END == eventNum) {
+		/* convert event from concurrent-end to scavenge-end */
+		MM_ScavengeEndEvent scavengeEndEvent;
+		MM_ConcurrentLocalPhaseEndEvent *event = (MM_ConcurrentLocalPhaseEndEvent *)eventData;
+		scavengeEndEvent.currentThread = event->currentThread;
+		scavengeEndEvent.timestamp = event->timestamp;
+		scavengeEndEvent.eventid = event->eventid;
+		scavengeEndEvent.subSpace = NULL; //unknown info
+		scavengeEndEvent.cycleEnd = false;
+
+		handleScavengeEndNoLock(hook, J9HOOK_MM_PRIVATE_CONCURRENT_LOCAL_PHASE_END, &scavengeEndEvent);
```

Comment:
`J9HOOK_MM_PRIVATE_CONCURRENT_LOCAL_PHASE_END` should be `J9HOOK_MM_PRIVATE_SCAVENGE_END`